pages/firebase-inputs.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+
+// import { setNewArticle } from '../firebase/api/articles';
+
+const FirebaseArticleInputs = () => {
+  const [isTitleDifferent, setIsTitleDifferent] = useState(false);
+
+  const newArticleHandler = (e) => {
+    e.preventDefault();
+    // const articleObj = { title: e.target[0].value, content: e.target[1].value };
+
+    // eslint-disable-next-line no-console
+    console.log(e.target);
+    // setNewArticle(articleObj);
+  };
+
+  const isTtileDifferentChangeHandler = (event) => {
+    setIsTitleDifferent(!isTitleDifferent);
+  };
+
+  return (
+    <>
+      <form className="m-5" onSubmit={newArticleHandler}>
```

Comment:
I think functions should start with a verb so - `handleNewArticle` maybe `addNewArticle` ?

Reply:
I like to use the word "handler" at the end of the name for the function that handles the action of the form. It differentiate the handler from other functions, because the similar name as "addNewArticle" is used many times - in API article.js, a page name...

Reply:
Sure. If you will stick to it